### ISSUE TITLE
[Android] Cant select images in the library

### ISSUE THREAD

**Issue by AdamGorkoz** [marcshilling/react-native-image-picker#58]
When i reach the photo gallery all the pictures are disabled , Any ideas why ?

**Comment by yfuks** [comment]
Hello, 

Can you test to put the folowing code in your android\app\src\main\AndroidManifest.xml and tell me if it's resolve your problem

```xml
<manifest ...>

<!-- Import permissions -->
<uses-permission android:name="android.permission.CAMERA" />
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
    <uses-feature android:name="android.hardware.camera"
                  android:required="true"/>
    <uses-feature android:name="android.hardware.camera.autofocus" />
<!-- -->
```

**Comment by turka50** [comment]
@yfuks  Actually i diagnosed the problem more closely , When i go to the choose from library option it opens the whole file system to browse around and all the images are disabled except for when accessing the pictures app , Can i make it so that the pictures app opens instead of this file system ?

**Comment by yfuks** [comment]
Well i think it's in your device preferences, you should have something like this if preference is not set : 

<img href="https://cloud.githubusercontent.com/assets/9282988/11658269/0e16cee8-9dc2-11e5-9b33-cee3be301475.png" width="50%">

Can you tell me your **API** version ?

But if you can't select an image from the file explorer it's a problem, i will investigate this issue.

**Comment by turka50** [comment]
Hmm might be , I get something like this : 
![image](https://cloud.githubusercontent.com/assets/10440278/11659611/d590fdcc-9dd1-11e5-8de2-58825bf507d3.png)

This is using API 22 , This is an emulator , i dont have a device to show you right now ill check the api tomorrow and update you. Anyway i can only select from the gallery.

**Comment by timzaak** [comment]
I use a device, the android version is 5.0, Both `take photos` and `Choose from Library` do not work.

**Comment by yfuks** [comment]
Hello, do not work but can we have more information ? It crash ? You can't do something in particular ?
Did you try to debug ?

**Comment by skyride99** [comment]
I get the same thing as AdamGorkoz, api 22, RN 17. The pictures are disabled and not selectable

**Comment by yfuks** [comment]
Will try to fix it as soon as possible.

**Comment by atom992** [comment]
+1

**Comment by yfuks** [comment]
fix in 5e926e0

Now selecting the library will open the default Gallery app

In the worst case you get something like this : 

<img title="Android" src="http://i.stack.imgur.com/yzg9e.png" width="50%">

Where both Gallery and Photos working.

Tested on API 22 - 19 - 16 for emulator and 18 with a real device.

The fix is coming with the 0.10.3 version
